src/Utilities/WellKnownTypes.cs

```diff
@@ -1,11 +1,24 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System;
 using Microsoft.CodeAnalysis;
 
 namespace Analyzer.Utilities
 {
     internal static class WellKnownTypes
     {
+        public const string SystemException = "System.Exception";
+        public const string SystemDiagnosticContractsContract = "System.Diagnostics.Contracts.Contract";
+        public const string SystemIDisposable = "System.IDisposable";
+        public const string SystemThreadingMonitor = "System.Threading.Monitor";
+        public const string SystemThreadingTasksTask = "System.Threading.Tasks.Task";
+        public const string SystemCollectionsICollection = "System.Collections.ICollection";
+        public const string SystemCollectionsGenericICollection = "System.Collections.Generic.ICollection`1";
+        public const string SystemCollectionsGenericIReadOnlyCollection = "System.Collections.Generic.IReadOnlyCollection`1";
+        public const string SystemRuntimeSerializationSerializationInfo = "System.Runtime.Serialization.SerializationInfo";
+        public const string SystemIEquatable1 = "System.IEquatable`1";
+        public const string SystemRuntimeSerializationFormattersBinaryBinaryFormatter = "System.Runtime.Serialization.Formatters.Binary.BinaryFormatter";
+        public const string SystemWebUILosFormatter = "System.Web.UI.LosFormatter";
         public static INamedTypeSymbol ICollection(Compilation compilation)
         {
             return compilation.GetTypeByMetadataName("System.Collections.ICollection");
```

Comment:
>"System.Collections.ICollection" [](start = 53, length = 32)

We should change all the methods in this type to now use const strings defined above.